Admin/Extension/MenuNodeReferrersExtension.php

```diff
@@ -21,20 +21,47 @@
  */
 class MenuNodeReferrersExtension extends AdminExtension
 {
+    /**
+     * @var string
+     */
+    private $formTab;
+
+    /**
+     * @var string
+     */
+    private $formGroup;
+
+    public function __construct($formGroup = 'form.group_menu', $formTab = 'form.tab_menu')
+    {
+        $this->formGroup = $formGroup;
+        $this->formTab = $formTab;
+    }
+
     public function configureFormFields(FormMapper $formMapper)
     {
+        if ($formMapper->hasOpenTab()) {
+            $formMapper->end();
+        }
```

Comment:
instead of using tabs we can just float them in a nice way. i work a lot with `->with(..., [class => 'col-sm-7])` and `->with(..., [class => 'col-sm-5])`. Important blocks on the left, informatial stuff on the right. Looks nicer then tabs.